app/models/provider.rb

```diff
@@ -56,8 +56,10 @@ def all_courses_open_in_current_cycle?(exclude_ratified_courses: false)
     open_in_find.any? && all?(&:open_on_apply?)
   end
 
-  def any_open_courses_in_current_cycle?
-    accredited_courses.or(courses).current_cycle.exposed_in_find.any?(&:open_on_apply?)
+  def any_courses_open_in_current_cycle?(exclude_ratified_courses: false)
+    courses_to_check = exclude_ratified_courses ? courses : accredited_courses.or(courses)
+
+    courses_to_check.current_cycle.exposed_in_find.any?(&:open_on_apply?)
```

Comment:
can we use the scope `open_on_apply` which covers both exposed in find and open in apply instead?courses_to_check.current_cycle.open_on_apply.any?

Reply:
Good call!